src/combat/combatant.js

```diff
@@ -1,3 +1,37 @@
 export default class YearZeroCombatant extends Combatant {
   // TODO https://gitlab.com/peginc/swade/-/blob/develop/src/module/documents/SwadeCombatant.ts
-}
\ No newline at end of file
+
+  async setCardValue(cardValue) {
+    return this.setFlag('yzec', 'cardValue', cardValue);
+  }
+
+  get cardValue() {
+    return this.getFlag('yzce', 'cardValue');
+  }
+
+  get cardString() {
+    return this.getFlag('yzce', 'cardString');
+  }
+
```

Comment:
Already scope creeping myself, but I stole my own code and reused it here. Could we add these accessors as well?: 

```js
	get fastAction() {
		return this.getFlag("yzce", "fastAction");
	}

	get slowAction() {
		return this.getFlag("yzce", "slowAction");
	}
```

Reply:
I was expecting to need to add something like this, was waiting for the sidebar to make sure we had all the flags needed

Reply:
Probably not the last flags related to that sidebar, just the ones I nicked from fbl. I'll come up with some config API for buttons to display in sidebar.

Reply:
sure. we will probably have something for hands and cards as well before it's all said and done